HowToVenv/README.md

```diff
@@ -0,0 +1,110 @@
+## Main information docs: 
+https://docs.python.org/3/library/venv.html
+
+https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment
+
+
+## Why to use virtual environment?
+To isolate environment from the global environments and having same Python pip libraries as the project obtains is what venv is mainly used for. 
+Sometimes the project may require specific libraries to work as syntax of the libraries may change or they are depended on another older libraries.
```

Comment:
Specific versions?

Reply:
Maybe one could also write some example/more explanations like:
- What is the difference if I do pip install inside a virtual environment contra without one?
- What are libraries? What is an environment? (Really hard questions to answer in a simple way, actually!)

Reply:
Good points to make bit more simpler level explanations. :)

Reply:
Not sure if the picture shows in commits, but at least there are some visuals about versions  :) 
https://github.com/Naapi/beginners-python-weather/tree/master/HowToVenv